templates/SwaggerUi/index.php

```diff
@@ -17,60 +17,39 @@
 <!-- HTML for static distribution bundle build -->
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title><?= $title ?> - Swagger UI</title>
-    <?= $this->Html->css('/swagger_ui/css/swagger-ui.css') ?>
-    <?= $this->Html->meta('icon', '/swagger_ui/img/favicon-32x32.png', ['sizes' => '32x32']) ?>
-    <?= $this->Html->meta('icon', '/swagger_ui/img/favicon-16x16.png', ['sizes' => '16x16']) ?>
-    <style>
-        html
-        {
-            box-sizing: border-box;
-            overflow: -moz-scrollbars-vertical;
-            overflow-y: scroll;
-        }
+    <head>
+        <meta charset="UTF-8">
+        <title><?= $title ?> - Swagger UI</title>
+        <?= $this->Html->css('/swagger_ui/css/swagger-ui.css') ?>
+        <?= $this->Html->meta('icon', '/swagger_ui/img/favicon-32x32.png', ['sizes' => '32x32']) ?>
+        <?= $this->Html->meta('icon', '/swagger_ui/img/favicon-16x16.png', ['sizes' => '16x16']) ?>
+    </head>
 
-        *,
-        *:before,
-        *:after
-        {
-            box-sizing: inherit;
-        }
+    <body>
+        <div id="swagger-ui"></div>
+        <?= $this->Html->script('/swagger_ui/js/swagger-ui-bundle.js', ['charset' => 'UTF-8']) ?>
+        <?= $this->Html->script('/swagger_ui/js/swagger-ui-standalone-preset.js', ['charset' => 'UTF-8']) ?>
+        <script src="./swagger-initializer.js" charset="UTF-8"> </script>
```

Comment:
This file is missing plus the way it is included will not work in all cases.
If the file is necessary, we need to include it as we do `swagger-ui-bundle.js` or `swagger-ui-standalone-preset.js`.

Reply:
May be the init script is the one below?